### PR TITLE
[kjobctl] Added create job command.

### PR DESCRIPTION
**Pull request by mbobrovskyi**
#### What type of PR is this?
/kind feature

#### What this PR does / why we need it:
Added create job command on kjobctl.

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: cmd/experimental/kjobctl/Makefile [hunk 1]
```diff
@@ -83,13 +90,14 @@ vet: ## Run go vet against code.
...
-test-e2e:
-	go test ./test/e2e/ -v -ginkgo.v
+test: verify gotestsum gomod-download envtest ginkgo ## Run tests.
+	# Unit tests
+	$(GOTESTSUM) --junitfile $(ARTIFACTS)/junit.xml -- $(GO_TEST_FLAGS) $(shell $(GO_CMD) list ./... | grep -v '/test/') -coverpkg=./... -coverprofile $(ARTIFACTS)/cover.out
+	# Integration tests
+	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" \
+	KUEUE_BIN=$(PROJECT_DIR)/bin \
+	ENVTEST_K8S_VERSION=$(ENVTEST_K8S_VERSION) \
+	$(GINKGO) $(GINKGO_ARGS) -procs=$(INTEGRATION_NPROCS) --race --junit-report=junit.xml --output-dir=$(ARTIFACTS) -v $(INTEGRATION_TARGET)
```
Comment (trasc) [inline cmd/experimental/kjobctl/Makefile:100]: The integration tests will overwrite junit.xml, let's  try to put the reports in different files.

Comment (mbobrovskyi) [reply cmd/experimental/kjobctl/Makefile:100]:
```suggestion
	$(GOTESTSUM) --junitfile $(ARTIFACTS)/junit.xml -- $(GO_TEST_FLAGS) $(shell $(GO_CMD) list ./... | grep -v '/test/') -coverpkg=./... -coverprofile $(ARTIFACTS)/cover.out
	# Integration tests
	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" \
	KUEUE_BIN=$(PROJECT_DIR)/bin \
	ENVTEST_K8S_VERSION=$(ENVTEST_K8S_VERSION) \
	$(GINKGO) $(GINKGO_ARGS) -procs=$(INTEGRATION_NPROCS) --race --junit-report=kjobctl-junit.xml --output-dir=$(ARTIFACTS) -v $(INTEGRATION_TARGET)
```
Like this?

Comment (trasc) [reply cmd/experimental/kjobctl/Makefile:100]: That is one way but maybe with more relevant naming (junit-unit.xml, junit-integration.xml). Or create some subdirs in $(ARTIFACTS)

File: cmd/experimental/kjobctl/pkg/cmd/create/create.go [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+	}
+
+	if o.DryRunStrategy != util.DryRunClient {
+		obj, err = o.createObject(ctx, clientGetter, obj)
+		if err != nil {
+			return err
+		}
+	}
+
+	return o.PrintObj(obj, o.Out)
```
Comment (trasc) [inline cmd/experimental/kjobctl/pkg/cmd/create/create.go:224]: Just a note for now, the `Interactive` mode should keep the stdin stdout  streams open.

File: cmd/experimental/kjobctl/pkg/cmd/testing/fake.go
Comment (trasc) [inline cmd/experimental/kjobctl/pkg/cmd/testing/fake.go:1]: Missing Copyright header

**Review by trasc — COMMENTED** [review]
just some nits, otherwise LGTM

File: cmd/experimental/kjobctl/pkg/cmd/testing/fake.go
Comment (mbobrovskyi) [reply cmd/experimental/kjobctl/pkg/cmd/testing/fake.go:1]: Thanks

File: cmd/experimental/kjobctl/Makefile (hunk 1 above)
Comment (mbobrovskyi) [reply cmd/experimental/kjobctl/Makefile]: Done

**Review by trasc — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**